src/components/KeyboardHandler.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+import React, {useEffect} from 'react';
+import mousetrap from 'mousetrap';
+
+export default function KeyboardHandler({children, onSave}) {
+  useEffect(() => {
+    mousetrap.bind('mod+s', () => {
+      onSave();
+      return false;
+    });
+
+    return () => {
+      mousetrap.unbind('mod+s');
+    };
+  }, [onSave]);
+
+  return <>{children}</>;
```

Comment:
Per IRL discussion, taking children here may give the impression that the keypresses are only captured if the children are focused, but that is not the case—better to make it a void component.